trainingRN/src/screens/todo/layout.js

```diff
@@ -1,19 +1,27 @@
 import React, {Component} from 'react';
 import { Text, View, Button, TextInput } from 'react-native';
+import { CheckBox } from 'react-native-elements';
+
+import { ADD, DELETE_ALL, ADD_NEW_TASK, X } from '../../constants/constant';
 import styles from './styles';
 
-function TodoLayout({ handleDelete, data, handleNew, setNewText }) {
+function TodoLayout({ onDelete, data, handleNew, setNewText, checkElem, onDeleteAll }) {
     return (
       <View style={styles.container}>
-        <Text style={styles.task}>Add new task</Text>
+        <Text style={styles.task}>{ADD_NEW_TASK}</Text>
         <TextInput style={styles.textInput} onChangeText={text => setNewText(text)}/>
-        <Button title="Add" onPress={handleNew}/>
+        <Button title={ADD} onPress={handleNew}/>
         {data.map(element => 
         <View style={styles.taskBox} key={element.key}>
```

Comment:
I'd extract this `View` as a component, since conceptually this makes a component.